lib/matplotlib/artist.py

```diff
@@ -1437,7 +1437,12 @@ def setp(obj, *args, **kwargs):
       >>> setp(lines, linewidth=2, color='r')        # python style
     """
 
-    insp = ArtistInspector(obj)
```

Comment:
I just saw the same thing with regular `grep -r`. Would you mind if I absorbed some of your changes into my commit in the meantime?